### DISCUSSION TITLE
continuous e09dacb

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1496] [General]
SV
Exception: ser.ARDUINOTC4temperature(): could not convert string to float: '# freeMemory0=460'@line 6268
0

**Comment by Terracotta-6** [comment]
![78f54d24d84b22e31c2127beb789cb1](https://github.com/artisan-roaster-scope/artisan/assets/149906183/837d249d-85a9-4be2-9425-e820f5008d66)
![e2fc369ca82369b5a421c60df0ed57c](https://github.com/artisan-roaster-scope/artisan/assets/149906183/5df3574a-23d8-4c21-a479-2d44f01c65c4)

**Reply by MAKOMO** [reply]
The `ARDUINOTC4temperature` error in your first comment is because the TC4 did not return a number for the first extra device channel. This issue must be on the TC4 side.

Your last image must be after restarting Artisan where a (background triggered) SV out of range of the manual SV slider got saved to the setting and on load just displayed. Fixed. Thanks as always for your excellent reports and all your testing work.

**Reply by Terracotta-6** [reply]
Have been fixed

**Comment by Terracotta-6** [comment]
File "artisanlib\pid_dialogs.py", line 1148, in setSV------------------
------artisanlib\pid_dialogs.py::setSV:1148
An unhandled exception occurred. Please report the problem on Github:
https://github.com/artisan-roaster-scope/artisan/issues
When reporting this issue, please include your settings file (export
via menu Help >> Save Settings) and the details below.
An entry has been written to the error log (menu Help>> Error).
Version: 2.10.3,revision: 5d2640f
2024-03-08,14:20:47------
----------: 'PID_DIgControl' object has no attribute 'pidDutySteps'
(line: 1148)
OK

**Reply by MAKOMO** [reply]
This one should be fixed by now too. Thanks for reporting!

**Reply by Terracotta-6** [reply]
Have been fixed
